pkg/debugapi/node_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
```

Comment:
this should be `debugapi_test`, the on line 12 `const nonExistingMode debugapi.BeeNodeMode`